T2-TECHNICAL-OVERVIEW.md

```diff
@@ -0,0 +1,137 @@
+# Tessel 2 Technical Overview
+This guide is intended to help code contributors understand how relevant system components of Tessel 2 work, where to find code and design files, and accelerate the development process through technical transparency. 
+- [T2 Github Repositories](#relevant-github-repositories)
+    - [Tools](#tessel-2-tools-) 
+    - [Device Code](#device-related-code)
+    - [Hardware Designs](#hardware-designs)
+    - [Module Code](#modules)
+- [System Architecture](#system-architecture)
+    - [Hardware Overview](#high-level-look-at-hardware)
+    - [Software Overview](#software-architecture)
+- [Code Deploy Walkthrough](#a-code-deploy-example)
+- [Getting Questions Answered](#questions-)
+- [Links to more information](#more-useful-links)
+
+## Relevant Github Repositories
+
+###Tessel 2 Tools:
```

Comment:
This split-out is a bit weird to me. I would look for CLI, firmware, OS all together as the core, then look for the hardware, then look for the tools